e2e/process-services-cloud/task-form-cloud-component.e2e.ts

```diff
@@ -168,60 +161,64 @@ describe('Task form cloud component', () => {
-
-        await widget.amountWidget().setFieldValue('Amount0mtp1h', '660');
-        await expect(await (await taskFormCloudComponent.getCompleteButton()).isEnabled()).toBe(true);
+        beforeEach(async () => {
+            await appListCloudComponent.goToApp(candidateBaseApp);
+        });
 
-    });
+        it('[C307032] Should display the appropriate title for the unclaim option of a Task', async () => {
+            await tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
+            await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(assigneeTask.entry.name);
+            await tasksCloudDemoPage.taskListCloudComponent().selectRow(assigneeTask.entry.name);
+            await expect(await taskFormCloudComponent.getReleaseButtonText()).toBe('RELEASE');
+        });
 
-    describe('Complete task - cloud directive', () => {
+        it('[C310142] Empty content is displayed when having a task without form', async () => {
+            await tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
+            await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(assigneeTask.entry.name);
+            await tasksCloudDemoPage.taskListCloudComponent().selectRow(assigneeTask.entry.name);
+            await taskFormCloudComponent.checkFormIsNotDisplayed();
+            await expect(await taskFormCloudComponent.getFormTitle()).toBe(assigneeTask.entry.name);
+            await taskFormCloudComponent.checkFormContentIsEmpty();
+            await expect(await taskFormCloudComponent.getEmptyFormContentTitle()).toBe(`No form available`);
+            await expect(await taskFormCloudComponent.getEmptyFormContentSubtitle()).toBe(`Attach a form that can be viewed later`);
+        });
 
-        beforeEach(async () => {
-            await appListCloudComponent.goToApp(candidateBaseApp);
+        it('[C310199] Should not be able to complete a task when required field is empty or invalid data is added to a field', async () => {
+            await tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
+            await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(formValidationsTask.entry.name);
+            await tasksCloudDemoPage.taskListCloudComponent().selectRow(formValidationsTask.entry.name);
+            await taskFormCloudComponent.checkFormIsDisplayed();
+            await taskFormCloudComponent.formFields().checkFormIsDisplayed();
+            await taskFormCloudComponent.formFields().checkWidgetIsVisible('Text0tma8h');
+            await taskFormCloudComponent.formFields().checkWidgetIsVisible('Date0m1moq');
+            await taskFormCloudComponent.formFields().checkWidgetIsVisible('Number0klykr');
+            await taskFormCloudComponent.formFields().checkWidgetIsVisible('Amount0mtp1h');
+
+            await expect(await (await taskFormCloudComponent.getCompleteButton()).isEnabled()).toBe(false);
```

Comment:
Maybe create a method like isCompleteButtonEnabled()

Reply:
do we need 3 awaits here?